build.env

```diff
@@ -32,7 +32,7 @@ HELM_VERSION=v3.1.2
 # minikube settings
 MINIKUBE_VERSION=v1.12.3
 KUBE_VERSION=v1.18.8
-VM_DRIVER=none
+VM_DRIVER=virtualbox
```

Comment:
Travis CI does not allow VMs to get started, so we default to `none` for that.

Reply:
Ah! got it.